packages/mediacenter/kodi/package.mk

```diff
@@ -270,6 +270,7 @@ post_makeinstall_target() {
   rm -rf $INSTALL/usr/share/kodi/addons/skin.estuary
   rm -rf $INSTALL/usr/share/kodi/addons/service.xbmc.versioncheck
   rm -rf $INSTALL/usr/share/kodi/addons/visualization.vortex
+  rm -rf $INSTALL/usr/share/kodi/system/certs/cacert.pem
```

Comment:
Would it not be easier to install the symbolic link:
```
ln -sf /run/libreelec/cacert.pem $INSTALL/usr/share/kodi/system/certs/cacert.pem
```
and overwrite `certs/cacert.pem` with the symbolic link?

Then you can drop the 4 lines below.

Reply:
Yes, the forced link is sufficient, thanks.